endogenous/config.yaml

```diff
@@ -0,0 +1,6 @@
+scenario:
+  simpl: # only relevant for PyPSA-Eur
+    - ''
+  clusters: # number of nodes in Europe, any integer between 37 (1 node per country-zone) and several hundred
+    - 45
```

Comment:
Maybe clustering could be reduced to 37 so that the development cycle (i.e. developing-running-testing) can be faster - this may be relevant when running in a slower computer.

If clustering is reduced to 37, the CSV file for transport (and others that will follow) has to regenerated with this mind number in mind, and versioned in this Git repository afterwards.